spec/controllers/events_controller_spec.rb

```diff
@@ -157,6 +157,17 @@
       end
     end
 
+    describe "GET #send_participants_email" do
+      before :each do
+        @user = FactoryGirl.create(:user_with_profile, role: :admin)
+        sign_in @user
+      end
+      it "should assign a new email to accepted applications as @email" do
+        get :send_participants_email, id: @event.to_param, session: valid_session, all: true, groups: [], users:[], sender: 'workshop.portal@hpi.de'
+        expect(assigns(:email)).to have_attributes(hide_recipients: false, recipients: @event.email_addresses_of_accepted_applicants, reply_to: 'workshop.portal@hpi.de', subject: '', content: '')
```

Comment:
Please avoid hardcoding the email addresses. Use `Rails.configuration.reply_to_address` instead.